Patch for Ubuntu 16.04 LTS with gtk+3.0-3.20.0

--- a/configure.ac
+++ b/configure.ac
@@ -27,14 +27,14 @@
 GLIB_REQUIRED=2.48.1
 
 # for gtk+3.0 >= 3.22
-GTK_REQUIRED=3.22.0
-GDK_REQUIRED=3.22.0
-AC_DEFINE_UNQUOTED(MENU_GTK_3_22_AT_POINTER(M,E), gtk_menu_popup_at_pointer (M, E), "for gtk-3.22 and up")
+# GTK_REQUIRED=3.22.0
+# GDK_REQUIRED=3.22.0
+# AC_DEFINE_UNQUOTED(MENU_GTK_3_22_AT_POINTER(M,E), gtk_menu_popup_at_pointer (M, E), "for gtk-3.22 and up")
 
 # for gtk+3.0 3.20.0
-# GTK_REQUIRED=3.20.0
-# GDK_REQUIRED=3.20.0
-# AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup (M, NULL, NULL, NULL, NULL, E->button.button, E->button.time), "for gtk versions older than 3.22")
+GTK_REQUIRED=3.20.0
+GDK_REQUIRED=3.20.0
+AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup (M, NULL, NULL, NULL, NULL, E->button.button, E->button.time), "for gtk versions older than 3.22")
 
 GTKSOURCEVIEW_REQUIRED=3.12.0
 # LIBPEAS_REQUIRED=1.7.0
