patch to downgrade to gtk3+3.20 used by Ubuntu 16.04 LTS
--- a/configure.ac
+++ b/configure.ac
@@ -16,8 +16,8 @@
 
 AC_DEFINE_UNQUOTED(GXSM_VERSION_NAME, "Red-Force-One","GXSM version identifier screen name")
 AC_DEFINE_UNQUOTED(GXSM_WIDGET_PAD, 5, "default widget padding (replaces obsolete GNOME_PAD)")
-AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup_at_pointer (M, E), "for gtk-3.22 and up")
-# AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup (M, NULL, NULL, NULL, NULL, E->button.button, E->button.time), "for gtk versions older than 3.22")
+# AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup_at_pointer (M, E), "for gtk-3.22 and up")
+AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup (M, NULL, NULL, NULL, NULL, E->button.button, E->button.time), "for gtk versions older than 3.22")
 AC_DEFINE_UNQUOTED(GXSM_RES_BASE_PATH_DOT,     "org.gnome.gxsm3",              "GSettings and Builder dot base path without leadiong or trailing slashes")
 AC_DEFINE_UNQUOTED(GXSM_RES_BASE_PATH,         "org/gnome/gxsm3",              "GSettings and Builder base path without leadiong or trailing slashes")
 AC_DEFINE_UNQUOTED(GXSM_RES_PREFERENCES_PATH,  "org/gnome/gxsm3/preferences",  "GSettings path for preferences")
@@ -31,12 +31,12 @@
 GLIB_REQUIRED=2.48.1
  
 # for gtk+3.0 >= 3.22
-GTK_REQUIRED=3.22.0
-GDK_REQUIRED=3.22.0
+# GTK_REQUIRED=3.22.0
+# GDK_REQUIRED=3.22.0
 
 # for gtk+3.0 3.20.0
-# GTK_REQUIRED=3.20.0
-# GDK_REQUIRED=3.20.0
+GTK_REQUIRED=3.20.0
+GDK_REQUIRED=3.20.0
 # AC_DEFINE_UNQUOTED(MENU_AT_POINTER(M,E), gtk_menu_popup (M, NULL, NULL, NULL, NULL, E->button.button, E->button.time), "for gtk versions older than 3.22")
 
 GTKSOURCEVIEW_REQUIRED=3.12.0
